pkg/config/once.go

```diff
@@ -1,20 +1,59 @@
 package config
 
-import "sync"
+import (
+	"sync"
+
+	"github.com/notaryproject/notation-go/config"
+)
 
 var (
-	instance     *File
-	instanceOnce sync.Once
+	// configInfo is the config.json data
+	configInfo *config.Config
+	configOnce sync.Once
+
+	// signingKeyInfo if the signingkeys.json data
+	signingKeysInfo *config.SigningKeys
+	signingKeysOnce sync.Once
 )
 
-// LoadOrDefaultOnce returns the previously read config file.
-// If previous config file does not exists, it reads the config from file
+// Config reflects the config.json file.
+type Config = config.Config
+
+// SigningKeys reflects the signingkeys.json file.
+type SigningKeys = config.SigningKeys
+
+// KeySuite is a named key suite.
+type KeySuite = config.KeySuite
+
+// ExternalKey contains the necessary information to delegate the signing operation to the named plugin.
+type ExternalKey = config.ExternalKey
+
+// X509KeyPair contains the paths of a public/private key pair files.
+type X509KeyPair = config.X509KeyPair
+
+//CertificateReference is a named file path.
+type CertificateReference = config.CertificateReference
```

Comment:
Curious why we don't call `github.com/notaryproject/notation-go/config` directly and rename this package to something like `configutil`?

Reply:
Thanks for your suggestion.